common-tsconfig.json

```diff
@@ -17,6 +17,9 @@
         "strict": true,
         "sourceMap": true,
         "strictPropertyInitialization": false,
-        "target": "ES2017"
+        "target": "ES2017",
+        "esModuleInterop": true,
+        "baseUrl": ".",
+        "composite": true
     }
 }
```

Comment:
I get a marker that a newline  is missing at the end of file. Could please add it?